source/rust_verify/src/attributes.rs

```diff
@@ -282,6 +284,9 @@ pub(crate) fn parse_attrs(attrs: &[Attribute]) -> Result<Vec<Attr>, VirErr> {
                 Some(box [AttrTree::Fun(_, arg, None)]) if arg == "nonlinear" => {
                     v.push(Attr::NonLinear)
                 }
+                Some(box [AttrTree::Fun(_, arg, None)]) if arg == "spinoff_z3" => {
```

Comment:
This isn't necessarily specific to Z3.  Should the option be named something like `spinoff_prover`?